lib/rails_performance/instrument/metrics_collector.rb

```diff
@@ -18,8 +18,8 @@ class MetricsCollector
       def call(event_name, started, finished, event_id, payload)
         event = ActiveSupport::Notifications::Event.new(event_name, started, finished, event_id, payload)
 
-        mount_url = RailsPerformance.mount_at || "/rails/performance/"

```

Comment:
I thought that `mount_at` will never be `nil` or blank. So not sure why there is the fallback to `"/rails/performance"`. But please do let me know and sorry if it is a mistake to remove it.